cmd/minikube/cmd/delete.go

```diff
@@ -90,17 +90,17 @@ func init() {
 
 func deleteContainersAndVolumes() {
 	delLabel := fmt.Sprintf("%s=%s", oci.CreatedByLabelKey, "true")
-	errs := oci.DeleteContainersByLabel(oci.Docker, delLabel)
+	errs := oci.DeleteContainersByLabel("env", oci.Docker, delLabel)
```

Comment:
what do u think about moving "env" to the oci package, and name it something like "oci.DefaultSudoPrefix" or "oci.Prefix"

Reply:
I'm not really sure what these const strings bring to the table, maybe I can read more somewhere ? Like definining Foo as "foo" and Bar as "bar"

But if we do, they are likely to be called oci.Env and oci.Sudo instead of something odd.